resources/css/bem/contest-judge-results-scores.less

```diff
@@ -4,14 +4,15 @@
 .contest-judge-results-scores {
   display: grid;
   align-items: center;
-  grid-gap: 5px;
+  gap: 5px;
   // category-name value
   grid-template-columns: minmax(auto, 120px) auto;
```

Comment:
unrelated but this probably should be `auto 1fr` otherwise this happens:

![](https://s.kohaku.love/2025/03/firefox_2025-03-26_12-29-04.png)